app/models/file_upload_manager.rb

```diff
@@ -1,4 +1,3 @@
-require 'english'
```

Comment:
This appears to have been unused. This module would not load on the remote machine and removing this line had no observable change to the test suite results.